src/components/message/message.scss

```diff
@@ -0,0 +1,57 @@
+@use "../../design-tokens/colors.yml";
+@use "../../mixins/focus";
+@use "../../design-tokens/font.yml";
+@use "../../design-tokens/sizes.yml";
+@use "../../mixins/ms";
+@use "../../mixins/unit";
+
+/**
+ * 1. Display message and close button in a centered row.
+ * 2. Push message and close button as far apart as possible.
+ * 3. Condense line height in case it breaks to multiple lines.
```

Comment:
This comment should include the _why_. The why isn't really "in case it breaks to multiple lines," the why is probably more like "Our default `line-height` may visually compete with this element's padding, so we condense it."